Resources/config/security_expressions.xml

```diff
@@ -27,7 +27,7 @@
 
     <services>
         <service id="security.extra.twig_extension" class="%security.extra.twig_extension.class%" public="false">
-            <argument type="service" id="security.context" />
+            <argument type="service" id="security.authorization_checker" />
```

Comment:
Isn't it safer to add ``on-invalid="ignore"`` in the case the core validation pass is executed before ``SecurityCompatibilityPass`` ?